js/impact.js

```diff
@@ -1578,7 +1578,7 @@ var GLPIImpact = {
                         }
                         if (targetValue != null && targetValue != '' && targetValue != '&nbsp;') {
                             tdTitle.innerText = propname;
-                            tdValue.innerText = _.unescape(targetValue);
+                            tdValue.innerText = $('<textarea />').html(targetValue).text();
```

Comment:
It should work:
```suggestion
                            tdValue.innerText = _.escape(targetValue);
```

Reply:
I try it but for example, status is displayed like "Other &#62; test" (&#62; = >)